client/r/rhawc/man/HawcClient.Rd

```diff
@@ -0,0 +1,11 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
do we need this file? if not, delete

Reply:
if we do need some sort of docs, then let's just keep it short and copy the the package title name and put a link to our hawc documentation, https://hawc.readthedocs.org/

Reply:
I do not know if this is 100% necessary, but it is a file that is automatically generated during package development (and updated every time i update the package) and as far as I know every package I've seen has one for all the functions. I believe it is what take the description information at the top of an .R file (i.e. lines that lead with #') and turns them into the help documentation that is available when you type ?function or help(function). Maybe you already knew all that, in which case I will look into how to get rid of it.

Reply:
nope, I wasn't sure. If we need it then we'll keep it. Can you update the description per my comment above?